js/tests/visual/scrollspy.html

```diff
@@ -28,6 +28,7 @@
             <a class="dropdown-item" href="#three" tabindex="-1">three</a>
           </div>
         </li>
+        <li class="nav-item"><a class="nav-link" href="#four">@four</a></li>
```

Comment:
Change this to something like "Selected when page scrolled to bottom", so as to explicitly give the test expectation.